docs/sdc.rst

```diff
@@ -0,0 +1,103 @@
+.. _sdc:
+
+Susceptibility Distortion Correction (SDC)
+------------------------------------------
+
+Introduction
+~~~~~~~~~~~~
+
+:abbr:`SDC (suceptibility-derived distortion correction) methods usually try to
+make a good estimate of the field inhomoneity map. 
+The inhomogeneity map is directly related to the displacement of 
+a given pixel :math:`(x, y, z)` along the
+:abbr:`PE (phase-encoding)` direction (:math:`d_\text{PE}(x, y, z)`) is
+proportional to the slice readout time (:math:`T_\text{ro}`)
+and the field inhomogeneity (:math:`\Delta B_0(x, y, z)`)
+as follows ([Jezzard1995]_, [Hutton2002]_):
+
+  .. math:: d_\text{PE}(x, y, z) = \gamma \Delta B_0(x, y, z) T_\text{ro}
+      :label: fieldmap
```

Comment:
Is there anyway to rewrite this so that the label goes on the same line, rather than above it? This just looks weird:

![weird](https://user-images.githubusercontent.com/83442/32813153-b2284066-c976-11e7-87ae-1a82c66e9eb5.png)

Reply:
I've tried several things, nothing worked. It's indeed very ugly and weird.

Reply:
Ok, using a full reference seems to look much better :+1: